adapter/service/chargepoint/specification.go

```diff
@@ -95,3 +102,27 @@ func requiredInterfaces() []fimptype.Interface {
 		},
 	}
 }
+
+// energyManagementInterfaces returns optional energy management interfaces of the service.
```

Comment:
If you change names for controller and support method, then change this one too.

Reply:
Done.